src/Data/Foldable.purs

```diff
@@ -177,6 +177,8 @@ fold = foldMap id
 -- |
 -- | Note: this function is not generally stack-safe, e.g., for monads which 
 -- | build up thunks a la `Eff`.
+-- | For a stack-safe alternative that works on lists, see
+-- | [purescript-safely](https://pursuit.purescript.org/packages/purescript-safely/3.0.0/docs/Control.Safely#v:foldM)
```

Comment:
This link will become outdated if `purescript-safely` ever makes a new release (like now during v0.14.0 migration phase).

Could we instruct people to search for `foldM` in the `purescript-safely` package instead?